hbase-http/src/main/java/org/apache/hadoop/hbase/util/JSONBean.java

```diff
@@ -125,11 +130,13 @@ public int write(MBeanServer mBeanServer, ObjectName qry, String attribute,
    */
   private static int write(JsonWriter writer, MBeanServer mBeanServer, ObjectName qry,
       String attribute, boolean description) throws IOException {
-    LOG.trace("Listing beans for " + qry);
+    LOG.debug("Listing beans for " + qry);
     Set<ObjectName> names = null;
     names = mBeanServer.queryNames(qry, null);
     writer.name("beans").beginArray();
     Iterator<ObjectName> it = names.iterator();
+    boolean pattern = false;
```

Comment:
You can avoid this boolean. Just use String[] and check for null value and call diff write methods?

Reply:
Ok